src/Controller/MenuController.java

```diff
@@ -195,6 +196,13 @@ public void main() {
 	 * Display the Study Dashboard pane.
 	 */
 	public void loadDashboard() {
+		
+		//set ToolTips
+		openMenu.setTooltip(new Tooltip("Menu"));
+		showNotification.setTooltip(new Tooltip("Notifications"));
+		addActivity.setTooltip(new Tooltip("Add activity"));
+		calendar.setTooltip(new Tooltip("Open Calendar"));
+		
```

Comment:
This trailing white space needs to be removed.

Reply:
This hasn't come up as trailing white space for me, but instead as simply tabbing.
<img width="376" alt="screenshot_20171001-222932" src="https://user-images.githubusercontent.com/25588487/31062358-195219d4-a6f8-11e7-9aec-56657fc62003.png">

Reply:
The blank lines above and below the block of code you added have trailing white space.

Reply:
I guess shouldn't the space after the text remain, for organizational purposes? In that screenshot, many other pre-existing blocks of code have the same trailing whitespace.

Reply:
I don't mean to remove the blank lines.  I mean leave the blank lines, but remove the trailing spaces/tabs on those lines.